Tests/ClientTests/TabManagerStoreTests.swift

```diff
@@ -135,8 +135,9 @@ private extension TabManagerStoreTests {
         manager.storeChanges(writeCompletion: { [weak self] in
             guard let self = self else { XCTFail("self should be strong"); return }
 
-            let managerMessage = "TestTabCountOnDisk is \(manager.testTabCountOnDisk()) when it should be \(managerTabCount)"
-            XCTAssertEqual(manager.testTabCountOnDisk(), managerTabCount, managerMessage, file: file, line: line)
+            // TODO: Improve upon store tests so we don't depend on testTabCountOnDisk()
+//            let managerMessage = "TestTabCountOnDisk is \(manager.testTabCountOnDisk()) when it should be \(managerTabCount)"
+//            XCTAssertEqual(manager.testTabCountOnDisk(), managerTabCount, managerMessage, file: file, line: line)
```

Comment:
I would just delete this and remove the TODO. If/when we get back to this area to improve the tests it will be easy to discern which test cases are lacking without needing a TODO as a reminder.